src/Makefile.am

```diff
@@ -85,6 +90,7 @@ libucc_la_SOURCES =                   \
 	core/ucc_version.c                \
 	core/ucc_context.c                \
 	core/ucc_mc.c                     \
+	core/ucc_ec.c                     \
```

Comment:
generic question (i didn't honestly understand that from day one): why ucc_mc.c(h) and ucc_ec.c(h) are in "core" and not under src/components/mc and src/components/ec respectively?

Reply:
agree